package.json

```diff
@@ -1,33 +1,70 @@
 {
-   "name": "fsaba",
-   "version": "0.2.1",
+   "name": "@silvermine/fsaba",
+   "version": "0.1.0",
```

Comment:
Not sure the plans for the version number after this PR, but it seems a little funny to go from v0.2.1 to v0.1.0. Of course the package name is changing so 🤷‍♂️